### PR TITLE
[Mint 3067]/be analytics queries

### PR DESCRIPTION
**Pull request by StevenWadeOddball**
# MINT-3067

## Description

This PR's goal is to create some basic queries to expose some  queries to provide basic analytics about  MINT data. Some data will need to be further modified as we proceed to fine tune it to the final tool we are using. This however will provide a starting point to answer the questions being asked.

You will note there is not a date rang on these queries currently, but look at data for all time. Also, many of these queries might be more useful when constrained to a single model at a time, but it is possible to run across the entire data set.

## How to test this change

1.  Run each query against the database.
2. Ensure that the queries function without error.

If desired, you can run this against a fuller data set. At the very least, I suggest that you seed the database before testing the queries

## PR Author Checklist

- [ ] I have provided a detailed description of the changes in this PR.
- [ ] I have provided clear instructions on how to test the changes in this PR.
- [ ] I have updated tests or written new tests as appropriate in this PR.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.

### CODE REVIEW

File: pkg/sqlqueries/SQL/analytics/number_of_followers_per_model.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* Number of Followers By Model */
+SELECT
+    model_plan.model_name,
```
Comment (ClayBenson94) [inline pkg/sqlqueries/SQL/analytics/number_of_followers_per_model.sql:3]: Would you mind adding `model_plan.id` here as well like we do in the other queries?

**Review by ClayBenson94 — CHANGES_REQUESTED** [review]
Looks good! Just one nit-pick on adding an extra field, but otherwise I'm ready to approve!

File: pkg/sqlqueries/SQL/analytics/number_of_followers_per_model.sql (hunk 1 above)
Comment (StevenWadeOddball) [reply pkg/sqlqueries/SQL/analytics/number_of_followers_per_model.sql:3]: Thanks @ClayBenson94 ! Added

**Review by ClayBenson94 — APPROVED** [review]
Changes look good -- thanks @StevenWadeOddball!

**Pull request merged**